discovery-provider/src/__init__.py

```diff
@@ -139,6 +150,7 @@ def create_celery(test_config=None):
     global playlist_factory
     global user_library_factory
     global ipld_blacklist_factory
+    global user_replica_set_manager
     global contract_addresses
```

Comment:
There are a lot of global variables here and on line 137 above, I think it might be nice to merge these into a single line.

Reply:
i think this is unrelated / out of scope but i agree